Automattic/wp-calypso
# Add Beta Testing fallback menu item

#### Changes proposed in this Pull Request

This adds the nav link for the Beta Testing page in the Fallback Menu. 
The page is added in PR #54346 and the nav link to it is added in D63456-code.

NOTE: #54346 and D63456-code should be deployed before merging this.

#### Testing instructions

* The Fallback Menu briefly appears before the response from the Public API enpoint /wpcom/v2/sites/{BLOG_ID}/admin-menu. You can block this URL to prevent the response from happening. You can also Clear Site Data to make sure the Fallback Menu is used.
* The Beta Testing link should appear as the last item, in the left Sidebar menu:
<img width="320" alt="Screenshot on 2021-07-15 at 14-00-18" src="https://user-images.githubusercontent.com/2749938/125777895-edbbdf2a-1417-40b4-9c98-58ba5ef89953.png">

* The link should not appear on P2 sites
* It should open the /beta-testing page
